healthcare/healthcare/doctype/inpatient_record/inpatient_record.js

```diff
@@ -11,28 +11,52 @@ frappe.ui.form.on('Inpatient Record', {
 		];
 	},
 	refresh: function(frm) {
-		if (!frm.doc.__islocal && (frm.doc.status == 'Admission Scheduled' || frm.doc.status == 'Admitted')) {
-			frm.enable_save();
-		} else {
-			frm.disable_save();
-		}
+		frm.set_query('admission_service_unit_type', function() {
+			return {
+				filters: {
+					'inpatient_occupancy': 1,
+					'allow_appointments': 0
+				}
+			};
+		});
 
-		if (!frm.doc.__islocal && frm.doc.status == 'Admission Scheduled') {
-			frm.add_custom_button(__('Admit'), function() {
-				admit_patient_dialog(frm);
-			} );
-		}
+		frm.set_query('primary_practitioner', function() {
+			return {
+				filters: {
+					'department': frm.doc.medical_department
+				}
+			};
+		});
+		if (!frm.doc.__islocal) {
+			if (!frm.doc.admission_encounter) {
```

Comment:
Club both if statements?

Reply:
yes thanks, I've made some minor changes
- If statements clubbed
- Moved 'Admit' button show/hide part to json display_depends_on